### DISCUSSION TITLE
I want to style the a inside the last li

### DISCUSSION THREAD

**Discussion by nehiro** [tailwindlabs/tailwindcss#11894] [Help]
Hi. As the title says, I want to style the a inside the last-child of li, please help😭
<img width="763" alt="スクリーンショット 2023-08-25 19 11 51" src="https://github.com/tailwindlabs/tailwindcss/assets/59185285/6e40e497-2fc5-430b-a421-613bf3ece1d7">

**Comment by wongjn** [comment]
Assuming that an `<a>` element is somehow rendered from the `<Link>` component and that this is React code:
- If `Link` is a custom component, consider checking the `className` prop is passed down to the `<a>` element that `Link` renders.
- If `Link` is a third-party component, consider checking the relevant documentation or reviewing the third-party code to see if there is a way to apply class names to the underlying `<a>` element that this `Link` component seemingly renders.

**Comment by nikolailehbrink** [comment]
Hi @nehiro,

You can refer to the relevant section of the Tailwind documentation for more details: [Tailwind CSS Documentation on Styling based on parent state](https://tailwindcss.com/docs/hover-focus-and-other-states#styling-based-on-parent-state).

I'm presuming that your `<Link>` component refers to the `next/link` component from Next.js. To achieve your desired effect, assign the `group` class to the list item (`<li>`) and then add the `group-last:text-purple-500` class to the `<Link>` component. This ensures that only the `<Link>` component in the last list item (`<li>`) gets the specified styling. I hope this clears things up for you!

```jsx
<ul>
  {links.map(item => {
    <li className="group">
      <Link className="group-last:text-red-500">{item.name}</Link>
    </li>
  })}
</ul>
```
